recompose-flow.md

```diff
@@ -69,42 +69,37 @@ const EnhancedComponent = enhance(baseComponent);
 ```
 
-You don't need to provide types for enhancers arguments, flow will infer them automatically.
+You don't need to provide types for arguments to enhancers; Flow will infer them automatically.
 
-Just remove `defaultProps` in the example above and you would immediately get a flow error
+Remove `defaultProps` in the example above and you would immediately get a Flow error:
 
 ![[flow] undefined (This type cannot be coerced to string)](./error.png?raw=true)
 
-The other wonderful feature is the flow ability to infer types automatically,
-this makes expirience with recompose even more better than ever.
-
-See this in action.
+The other wonderful feature is Flow's ability to infer types automatically,
+which makes the Recompose experience even better.  See it in action:
 
 ![recompose-flow](https://user-images.githubusercontent.com/5077042/28116959-0c96ae2c-6714-11e7-930e-b1454c629908.gif)
 
-For me this feature is much more cool than error detection, as
-it allows to read and understand code much more faster.
-
-Be sure all the magic above works fine for the most of recompose enhancers, but not for all. Any type system has it's own limitations, so for some enhancers you need to provide type information for every special case (no automatic type inference) or use our recommendations.
+For me this feature is much cooler than error detection, as
+it allows you to read and understand the code much faster.
 
-These are:
+The magic above works for most Recompose enhancers, but not all. Any type system has its limitations, so for some enhancers you will need to provide type information for every special case (no automatic type inference), or use the following recommendations:
 
-- `flattenProp`,`renameProp`, `renameProps` - recommendation is to replace with `withProps` in your code
+- `flattenProp`, `renameProp`, `renameProps` --- replace with `withProps` 
 
-- `withReducer`, `withState` use `withStateHandlers`
+- `withReducer`, `withState` --- use `withStateHandlers`
 
-- `lifecycle` write your own enhancer instead, see the example below.
+- `lifecycle` --- write your own enhancer instead; see [this test](https://github.com/acdlite/recompose/blob/master/types/flow-typed/recompose_v0.24.x/flow_v0.53.x-/test_mapPropsStream.js) for an example
 
-- `mapPropsStream` see the [test](https://github.com/acdlite/recompose/blob/master/types/flow-typed/recompose_v0.24.x/flow_v0.53.x-/test_mapPropsStream.js) for example
+- `mapPropsStream` --- see the [test](https://github.com/acdlite/recompose/blob/master/types/flow-typed/recompose_v0.24.x/flow_v0.53.x-/test_mapPropsStream.js) for an example
 
-As an example how to type such enhacers see [test](
-https://github.com/acdlite/recompose/blob/master/types/flow-typed/recompose_v0.24.x/flow_v0.53.x-/test_voodoo.js)
+For an example of how to type enhancers like this, see [this test](https://github.com/acdlite/recompose/blob/master/types/flow-typed/recompose_v0.24.x/flow_v0.53.x-/test_voodoo.js).
```

Comment:
You have "enhancers" spelled "enhacers" in a few places in your test code --- I guess the code still compiles because you used the same spelling consistently, but if you're going to make the tests public and link to them, you might consider correcting the spelling. 

You also have "errors" spelled "erros" here: https://github.com/acdlite/recompose/blob/b82aa76b1f83d7c00c771e4e0919ebb459ea6b07/types/flow-typed/recompose_v0.24.x/flow_v0.53.x-/test_mapPropsStream.js#L29